tsl/test/sql/compression.sql

```diff
@@ -1239,3 +1239,21 @@ where uncompressed.compressed_chunk_id = compressed.id AND uncompressed.id = :'C
 
 SELECT _ts_meta_count FROM :COMPRESSED_CHUNK_NAME ORDER BY device, _ts_meta_min_1 DESC;
 ROLLBACK;
+
+-- Test poor compression rate warning works as expected
+
+-- Turn GUC on
+SET timescaledb.enable_compression_ratio_warning TO ON;
+
+-- Compressing a table with very few rows virtually guarantees a poor compression rate
+CREATE TABLE badly_compressed_ht (time timestamptz, device_id integer, a integer);
+SELECT create_hypertable('badly_compressed_ht', 'time');
+ALTER TABLE badly_compressed_ht set (timescaledb.compress, timescaledb.compress_segmentby = 'device_id');
+INSERT INTO badly_compressed_ht VALUES
+('2025-04-25 00:00'::timestamp, 1, 1),
+('2025-04-25 01:00'::timestamp, 2, 2),
+('2025-04-25 02:00'::timestamp, 3, 3);
+
+SELECT compress_chunk(show_chunks('badly_compressed_ht'));
```

Comment:
Might be good to add a comparison with what `pg_column_size` reports on the compressed chunk, i.e. bytes-per-row ratio:
```
select count(*), sum(pg_column_size(clientip))::numeric / sum(_ts_meta_count) from :chunk;
```

To find the compressed chunk table: 
```

select schema_name || '.' || table_name chunk from _timescaledb_catalog.chunk
    where id = (select compressed_chunk_id from _timescaledb_catalog.chunk
        where hypertable_id = (select id from _timescaledb_catalog.hypertable
            where table_name = 'badly_compressed_ht') limit 1)
\gset
```

Reply:
>  Can you elaborate on how this comparison helps?

It's a query that our users can use to investigate the compression ratio in existing tables w/o recompression, so might be good to make sure that it shows similar results. Might be not very important though.